python/ray/data/_internal/arrow_block.py

```diff
@@ -187,7 +187,7 @@ def _get_max_chunk_size(
     if table.nbytes == 0:
         return None
     else:
-        avg_row_size = int(table.nbytes / table.num_rows)
+        avg_row_size = max(int(table.nbytes / table.num_rows), 1)
```

Comment:
Let's actually just remove the rounding

```suggestion
        avg_row_size = table.nbytes / table.num_rows
```

Reply:
Happy to do whatever but when int casts a small number we get zero causing the issue addressed in the PR